### ISSUE TITLE
Visual bug: success messages in the onboarding flow are hard to follow

### ISSUE THREAD

**Issue by Daniel-Warner-X** [getappmap/vscode-appland#582]
The layout for success messages are inconsistent in the onboarding flow. This makes it hard to understand when it is time for the user to press 'Next' which is a critical step for onboarding. Success messages should be clear, evident, and consistent so that the user knows where to look to see the status of the system.

For example the step one success message looks like this: 

![Image](https://user-images.githubusercontent.com/1229326/211093354-f1a3b19f-4993-4e59-bc7c-a9db3896ae60.png)

The step two success message looks like this:

![Image](https://user-images.githubusercontent.com/1229326/211093517-30c49aa5-4d13-4f8a-afda-aa5778a842c8.png)

Neither one looks particularly like a status message.

**Comment by kgilpin** [comment]
Yeah. Also a key phrase is "You're good to go" and it's not prominent.
